error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -221,6 +221,14 @@ ImmutableSet<Flux<String>> testMonoFlux() {
         Flux.concat(Mono.just("baz")));
   }
 
+  Mono<Void> testMonoFluxThen() {
+    return Mono.just("foo").flux().then();
+  }
+
+  Mono<Void> testMonoVoidThen() {
+    return Mono.just("foo").then().then();
```

Comment:
The most concise way I could think of creating a `Mono<Void>`, not the prettiest

Reply:
How about `Mono.<Void>empty()`? :)